src/rabin.rs

```diff
@@ -45,8 +45,9 @@ lazy_static! {
 ///
 /// ```rust
 /// use byteorder::{ByteOrder, LittleEndian};
+/// use hex_literal::hex;
 ///
-/// let i = LittleEndian::read_i64(&result.to_vec())
```

Comment:
Why this change?

Reply:
Ah, I see now! The test for the docs are broken and I haven't noticed because of https://github.com/rust-lang/cargo/issues/6669. Let me fix the tests first, but feel free to ignore this one.

Reply:
I fixed the docs test, please pull master.

Reply:
Done